include/openthread/border_agent.h

```diff
@@ -299,6 +299,20 @@ void otBorderAgentSetEphemeralKeyCallback(otInstance                       *aIns
                                           otBorderAgentEphemeralKeyCallback aCallback,
                                           void                             *aContext);
 
+/**
+ * Starts the Border Agent service.
+ *
+ * @param[in] aInstance    The OpenThread instance.
+ */
+void otBorderAgentStart(otInstance *aInstance);
+
+/**
+ * Stops the Border Agent service.
+ *
```

Comment:
Can help to mention the behavior when there is an already connected commissioner (candidate), i.e. the connection will be terminated?

Reply:
Done